### DISCUSSION TITLE
Create a feed from a Facebook Groups page?

### DISCUSSION THREAD

**Discussion by tomhusband** [FreshRSS/FreshRSS#6585] [Q&A] [answered]
I'd like to create a feed from a local Facebook Groups page but can't seem to find a way. I think Facebook makes it pretty tough but is there a way?

**Comment by Alkarex** [comment, marked as answer]
No for anything that requires a login.
More generally: are you able from the command line to curl the page or API with the relevant data? If yes, maybe, if no, then no.
